app/views/admin/dashboard/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<h1>Admin Dashboard</h1>
```

Comment:
Still think there is a case for using a partial

Reply:
We're adding a button to the packaged orders, maybe it would be useful to use a partial for the other three sections